src/v3/global.css

```diff
@@ -918,3 +918,9 @@
     box-shadow: 0px 2px 8px 0px #0e1227a3;
   }
 }
+
+
+.pop-up{
```

Comment:
@eloyEntony: would it be possible to place this additional code in another file, may be the `index.css`? Since this `global.css` is auto-generated, and if later we need to regenerate it, all manual entries will all be lost.